packages/esm-patient-list-app/src/api/api-remote.ts

```diff
@@ -136,8 +136,19 @@ export async function addPatientToList(data: AddPatientData, ac = new AbortContr
   return postData(`${cohortUrl}/cohortmember`, data, ac);
 }
 
-export async function createPatientList(cohort: NewCohortDataPayload, ac = new AbortController()) {
+export async function removePatientFromList(cohortMembershipUuid: string, ac = new AbortController()) {
   return postData(
+    `${cohortUrl}/cohortmember/${cohortMembershipUuid}`,
+    {
+      endDate: new Date(),
+      voidReason: '',
+    },
+    ac,
+  );
+}
+
+export async function createPatientList(cohort: NewCohortDataPayload, ac = new AbortController()) {
+  return deleteData(
```

Comment:
```suggestion
  return postData(
```

Reply:
@ibacher and @vasharma05 thanks for the review let share the proposed changes shortly!